index.html

```diff
@@ -88,13 +88,13 @@ <h2>The developers community<br>in Crete</h2>
 
   <!-- Upcomign Events -->
 	<div id="events">
-	  <div class="event" ng-if="MeetupList[0].IsFutureMeetup==1">
+	  <div class="event" ng-repeat="meetup in MeetupList" ng-if="isFutureMeetup(meetup.Day, meetup.Month, meetup.Year)">
```

Comment:
This block needs 1 level deeper identation

Reply:
Fixed at the latest commit.